client/src/components/boxwrapper/BoxWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
Можно сразу импортировать `Fragment`

```suggestion
import React, {Fragment} from "react";
```